pkg/kubelet/kubelet_node_status.go

```diff
@@ -976,7 +976,6 @@ func (kl *Kubelet) validateNodeIP() error {
 		var ip net.IP
 		switch v := addr.(type) {
 		case *net.IPNet:
-			ip = v.IP
```

Comment:
Golang doesn't fallthrough. By removing this line, you're ignoring all `net.IPNet` addressses, which I don't think is the intent?